api/routes/subcriterias/routes.py

```diff
@@ -32,10 +33,15 @@ def get(self, assessment_id: str = None, sub_criteria_id: str = None):
         :return: 200 assessment JSON / 404 Error
         """
         try:
-            sub_criteria = self.get_by_id(sub_criteria_id)
-        except SubCriteriaError as e:
-            return error_response(404, e.message)
-        except StatementError:
-            return error_response(404, "Sub-Criteria could not be found")
+            subcriteria = self.get_by_id(sub_criteria_id)
+        except SubCriteriaError:
+            return error_response(
+                code=404,
```

Comment:
rename this "code" occurance and all others to "status" as to conform to naming convention set in line 22 of same file

Reply:
Code is set in the error_response function, while line 22 uses another response function.